src/pages/CreateEvent.js

```diff
@@ -0,0 +1,118 @@
+// src/components/ProfileForm.js
+import React, { useState } from "react";
+import "./CreateEvent.css"; // Import the CSS file
+
+const locations = ["Copenhagen", "Aarhus", "Odense", "Aalborg"];
+const petTypes = ["Dog", "Cat", "Bird", "Other"];
+
+const ProfileForm = () => {
+  const [location, setLocation] = useState("");
+  const [headline, setHeadline] = useState("");
+  const [description, setDescription] = useState("");
+  const [datetime, setDatetime] = useState("");
+  const [eventPetType, setEventPetType] = useState("");
+
+  const handleLocationChange = (event) => {
+    setLocation(event.target.value);
+  };
+
+  const handleSubmit = () => {
```

Comment:
Consider adding basic validation and feedback for user input, such as checking the length of the username or ensuring passwords match.